views/admin/en/wirecardpaymentgateway_en_lang.php

```diff
@@ -12,6 +12,10 @@
 $aLang = array(
     "yes" => 'Yes',
     "no" => 'No',
+    "SHOP_MODULE_blEmailOnPending" => "Send notification emails when order pending",
```

Comment:
why UPPERCASE?

Reply:
it is translate key generated by OXID when you add some settings to the plugin metadata.php

Reply:
@ps-ohanel Can you elaborate a bit more? Does it **need** to be like this?

Reply:
Just to clarify: when defining module settings in metadata.php, OXID takes your lowercase defined names and prefixes it with these uppercase strings in auto generated FORM.